packages/patternfly-3/patternfly-react/src/components/TypeAheadSelect/TypeAheadSelect.js

```diff
@@ -19,4 +20,6 @@ TypeAheadSelect.asyncContainer = asyncContainer;
 TypeAheadSelect.menuItemContainer = menuItemContainer;
 TypeAheadSelect.tokenContainer = tokenContainer;
 
+TypeAheadSelect.Async = AsyncTypeAheadSelect;
+
```

Comment:
Note: Unsure if we want the `GithubMenuItem` exposed...

Reply:
I don't think so, its only used in the storybook.